tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -2014,3 +2016,117 @@ def test_functions_layers_with_s3_codeuri(self):
                 [""],
                 command_result,  # there is only one stack
             )
+
+
+class TestBuildWithCDKPluginNestedStacks(CDKTestBase):
```

Comment:
I'd suggest we move this test case to a new file to avoid any potential merge conflicts as we merge `develop` into the `cdk-support-dev` branch